standards/reserving-infrastructure-capacity.md

```diff
@@ -0,0 +1,75 @@
+---
+category: Operating services
+---
+# Reserving infrastructure capacity
+
+To get the best value for money from cloud infrastructure, we should
+pay in advance for parts of it that are going to be in active use over
+a long period.
+
+This guidance should be followed wherever possible, but is particularly
+applicable to
+[AWS](https://aws.amazon.com/ec2/pricing/reserved-instances/) and
+[Azure](https://azure.microsoft.com/en-us/pricing/reserved-vm-instances/).
+
+## Principles
+
+### Only reserve instances for long-lived infrastructure
+
+Reserving infrastructure allows us to make a significant saving over
+paying for it on demand, but only if that class of infrastructure is in
+use over the period in which the savings are made.
+
+The best indicator that we have of this is if that class of
+infrastructure has been (and, therefore, is likely to continue to be)
+in active use for the length of the reservation.
+
+Given that most infrastructure reservation systems operate on a one- or
```

Comment:
There _may_ be cases where it is reasonably obvious that we will need infrastructure from day 1 that will be around for e.g. a year (I'm thinking things moving over from other contracts/providers). I don't think it's a big point but are you happy to exclude these from consideration for reserved instances?